backend/core/models.py

```diff
@@ -1310,6 +1310,21 @@ class Status(models.TextChoices):
         DEPRECATED = "deprecated", _("Deprecated")
         UNDEFINED = "--", _("Undefined")
 
+    PRIORITY = [
+        ("P1", _("P1")),
+        ("P2", _("P2")),
+        ("P3", _("P3")),
+        ("P4", _("P4")),
+    ]
+
+    priority = models.CharField(
+        max_length=20,
+        choices=PRIORITY,
+        null=True,
+        blank=True,
+        verbose_name=_("Priority"),
+    )
```

Comment:
This is not compliant with the specs. Priority has to be an integer, "P1", "P2"... are only labels.